tests/codemods/test_exception_without_raise.py

```diff
@@ -0,0 +1,46 @@
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from core_codemods.exception_without_raise import ExceptionWithoutRaise
+from textwrap import dedent
+
+
+class TestExceptionWithoutRaise(BaseCodemodTest):
+    codemod = ExceptionWithoutRaise
+
+    def test_name(self):
+        assert self.codemod.name() == "exception-without-raise"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        ValueError
+        """
+        expected = """\
+        raise ValueError
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_alias(self, tmpdir):
+        input_code = """\
+        from libcst import CSTValidationError as error
```

Comment:
This only works because `libcst` is a dependency of codemodder itself, correct? We won't be able to detect custom exception types that are defined within the project under analysis and/or third-party dependencies of that project?

If Sonar is smarter than us then we can (eventually) use their report to fix the issues that we can't ourselves detect.

Reply:
> This only works because libcst is a dependency of codemodder itself, correct? 

Yes and I see that as a feature.

> We won't be able to detect custom exception types that are defined within the project under analysis and/or third-party dependencies of that project?

We can't really detect if a particular name is a class that derives from `BaseException` unless we somehow parse its declaration. This is unlikely to happen as we only use the module itself as context. Also, looking at sonar's implementation of the rule, it seems to only check for builtin exceptions